circuits/package.json

```diff
@@ -12,29 +16,32 @@
     "download": "bash scripts/server/download_circuits_from_AWS.sh",
     "format": "prettier --write .",
     "nice": "prettier --write .",
-    "install-circuits": "cd ../common && yarn && cd ../circuits && yarn",
+    "install-circuits": "yarn workspaces focus @selfxyz/circuits",
     "lint": "prettier --check .",
```

Comment:
should we add `--write` to this command? the github lint step is complaining

<img width="685" alt="Screenshot 2025-06-03 at 11 51 08 AM" src="https://github.com/user-attachments/assets/a5097330-3a93-4354-a20e-2892acee8976" />

Reply:
n/m missed the `format` command. we just need to run format